Gruntfile.js

```diff
@@ -38,12 +38,26 @@ module.exports = function(grunt) {
 
   grunt.registerTask('build', ['clean:dist', 'sass', 'webpack']);
 
+  // Regular grunt server, used when running inside the payments-env docker.
   grunt.registerTask('serve',
-    ['clean:dist', 'sass:dev', 'webpack-dev-server:start', 'watch:sass']);
-  grunt.registerTask('start',
     ['clean:dist', 'sass:dev', 'webpack:dev', 'watch:sass']);
-  grunt.registerTask('start-email',
+  // The dev server used for live reloading in the browser.
+  grunt.registerTask('serve:dev',
```

Comment:
This is a bit ambiguous as 'mytask:bar' would normally mean the bar target of the mytask task [1]

[1] http://gruntjs.com/creating-tasks#multi-tasks